Readme.md

```diff
@@ -56,6 +56,63 @@ Similarly, you can execute `npm run dotnet-postclean --if-present` via the `dotn
 To install *pnpm* globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
 
 
+## Global NPM vs Userspace NPM via Node Version Manager on Linux
+
+If you installed NPM from your package manager, you will likely suffer an `EACCES` error when you try to install a package globally. The [recommended solution](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#reinstall-npm-with-a-node-version-manager) is to install NPM via the Node Version Manager (NVM). This is what you should do:
+
+Before we start, ensure your directories are set up:
+```shell
+[ -z "$XDG_CONFIG_HOME" ] && echo 'export XDG_CONFIG_HOME="$HOME/.config"' >> ~/.bashrc && source ~/.bashrc
+mkdir -p "$XDG_CONFIG_HOME"
+mkdir -p "$HOME/.local/bin"
+```
+
+Also if you have installed npm globally via your package manager, uninstall it to avoid future confusion.
```

Comment:
Let's be consistent with the capitalization - in [line 61](#diff-1550ec65ac92f65817fc28928dfef526912b5f52356ff43651369bae92f56031R61) we have NPM, here npm, and later `npm`. I'd use the same everywhere, and prefer `npm`.

Ths same is true for `nvm` and `pnpm`.

Reply:
I have changed it here to "NPM". Whenever these words are not wrapped with backticks, they refer to the product rather than the command. So I think that's a meaningful distinction and not inconsistency.
As for pNpM, you can capitalize it however you want. Check out https://pnpm.io/ and see what happens as you press <kbd>F5</kbd> a few times. But for peace of mind, I have also replaced those with "PNPM" wherever they aren't backticked.

Reply:
That looks good to me.